### ISSUE TITLE
Seems I got permanent banned....

### ISSUE THREAD

**Issue by HenghaoZhou** [finndev/PokeBuddy#1114]
For two days showing me the message"Failed to get data from server"...Be careful guys..

**Comment by Abismo** [comment]
Me too... I got this message on 3 accounts and only one of them is still banned, for 48+ hours now.

**Comment by HenghaoZhou** [comment]
Did you get any email form Niantic?

**Comment by Abismo** [comment]
No, I didn't receive anything. And my pokemon are still at the gym I took a few days ago, I try to keep it with my other accounts just to see if my pokemon disappears.
